### PR TITLE
LPS-159906, LPS-163424, LPS-163427, LPS-163428, LPS-159637, LPS-160038, LPS-163702

### PR DESCRIPTION
**Pull request by thaiscabrals**
https://issues.liferay.com/browse/LPS-159906
https://issues.liferay.com/browse/LPS-163424
https://issues.liferay.com/browse/LPS-163427
https://issues.liferay.com/browse/LPS-163428
https://issues.liferay.com/browse/LPS-159637
https://issues.liferay.com/browse/LPS-160038
https://issues.liferay.com/browse/LPS-163702

### CODE REVIEW

**Comment by thaiscabrals** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectPortlet.testcase [hunk 1]
```diff
@@ -2566,6 +2566,18 @@ definition {
...
+
+		ObjectPortlet.assertEntryNotPresent(entry = "Test text 6");
+
+		ObjectPortlet.assertEntryNotPresent(entry = "Test text 7");
+
+		ObjectPortlet.assertEntryNotPresent(entry = "Test text 8");
+
+		ObjectPortlet.assertEntryNotPresent(entry = "Test text 9");
+
+		ObjectPortlet.assertEntryNotPresent(entry = "Test text 10");
```
Comment (andreefarias) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectPortlet.testcase:2580]: You can use a loop **(for)** for this case. It's will be easier to read or refactor your code.

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectPortlet.testcase [hunk 2]
```diff
@@ -2605,13 +2617,13 @@ definition {
...
+		ObjectPortlet.assertEntryNotPresent(entry = "Test text 011");
 
-		ObjectPortlet.assertEntryNotPresent(entry = "Test text 6");
+		ObjectPortlet.assertEntryNotPresent(entry = "Test text 012");
 
-		ObjectPortlet.assertEntryNotPresent(entry = "Test text 7");
+		ObjectPortlet.assertEntryNotPresent(entry = "Test text 013");
 
-		ObjectPortlet.assertEntryNotPresent(entry = "Test text 8");
+		ObjectPortlet.assertEntryNotPresent(entry = "Test text 014");
```
Comment (andreefarias) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectPortlet.testcase:2626]: Use loops here too.

**Comment by andreefarias** [comment]
**Good to go,** good job @thaiscabrals!

I will forward this PR, because it's a Test Fix.

**Comment by andreefarias** [comment]
ci:forward

**Pull request closed without merging**